### DISCUSSION TITLE
Config AUTH_DISABLE_DEFAULT should disable login form

### DISCUSSION THREAD

**Discussion by diegoleme** [directus/directus#11110] [Feature Requests]
Set `AUTH_DISABLE_DEFAULT` to true, should disable login form?

**Comment by joselcvarela** [comment]
This flag was created for this needs https://github.com/directus/directus/discussions/9298
In other words, you cannot disable the login form. But you can use this flag to login using other authentication mechanism than local.

The same happens with every flow that uses authentication, like inviting people.

However, you can take a look into shareable documents on right side bar, inside an item.

**Reply by diegoleme** [reply]
Sorry, but it's not clear to me what the `AUTH_DISABLE_DEFAULT` configuration does. Neither in the #9298 nor in the documentation... 

Can we explain this further in the documentation?

**Reply by joselcvarela** [reply]
cc @erondpowell 
I believe this flag should be used to prevent authentication using email and password.
This is useful for use-cases like LDAP or OpenID where you want your users to exclusively login via an external provider.

**Reply by diegoleme** [reply]
But in that case shouldn't the login form (email and password) be disabled?

**Reply by rijkvanzanten** [reply]
> But in that case shouldn't the login form (email and password) be disabled?

Yes it should. Mind opening a new [issue](https://github.com/directus/directus/issues/new)?

**Reply by diegoleme** [reply]
@rijkvanzanten I open #11114 and #11115.

**Reply by erondpowell** [reply]
Looks like this was more of a bug and the PR on this is closed??
Is it still relevant to note and potentially update docs?

**Reply by joselcvarela** [reply]
I believe we should mention this by any manner on docs.
Currently, we only have this regarding this flag
https://github.com/directus/directus/blob/ee7e678f24fd85846798bc3cc1af3103e728ebb7/docs/configuration/config-options.md#L556-L559

**Reply by NilsBaumgartner1994** [reply]
Definetly

**Reply by erondpowell** [reply]
Will open a PR to improve this later today.
